config/locales/service_down.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  service_down:
+    title: Sorry, there’s a problem with the service
+    try_later: Try again later.
+    progress_not_saved: If you reached this page after submitting information then it has not been saved. You’ll need to enter it again when the service is available.
+    questions_intro: If you have any questions, please email us at
+    questions_email: becomingateacher@digital.education.gov.uk
```

Comment:
I think we should rename the `service_unavailable.yml` to `error_pages.yml` and move all error page related locales in it, so they are easier to find and be shared across the error pages.

Reply:
Have changed the approach, so this no longer applies. Not entirely sure what we gain from having these translations in the first place for the service unavailable page, on which this was based.

Reply:
The translations are used by the [service_unavailable component](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/components/shared/service_unavailable_component.html.erb) which is rendered by the [service_unavailable middlewear](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/middlewares/service_unavailable_middleware.rb#L26).